src/nvim/keymap.c

```diff
@@ -282,7 +282,7 @@ static struct key_name_entry {
   {K_ZERO,            (char_u *)"Nul"},
   {K_SNR,             (char_u *)"SNR"},
   {K_PLUG,            (char_u *)"Plug"},
-  {K_PASTE,           (char_u *)"Paste"},
+  {K_PASTEPOST,       (char_u *)"PastePost"},
```

Comment:
See `tui/input.c` for why we don't need a "PastePre" pseudo-key. I tried to avoid "PastePost", but couldn't (and have some questions about that).